extension.js

```diff
@@ -54,6 +55,7 @@ function init() {
   // gnome 3.38 includes syntax changes to creating new overlay. 
   // this will be used later to decide which method to use in createOverlay 
   gnome_at_least_3_38 = isVersionGreaterOrEqual(3, 38);
+  gnome_at_least_40_1 = isVersionGreaterOrEqual(40, 1);
```

Comment:
The gnome_at_least_40_1 variable should be defined in prefs.js since the GTK api change happens there.
In prefs.js it would be like:
```
if (gnome_at_least_40_1) {
  ModifierKeyLabel.set_wrap(true); //For GTK4
} else {
  ModifierKeyLabel.set_line_wrap(true); // For GTK3
}
```
Then it would be compatible with both GTK3 and GTK4

Reply:
Ok, I will fix it.

Reply:
done